services/QuillCMS/docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
     environment:
       POSTGRES_USER: 'root'
       POSTGRES_PASSWORD: 'password'
+      discovery.type: single-node
```

Comment:
This doesn't seem necessary, since we use Postgres.app instead of docker now, but it also doesn't hurt anything.